doc/basics.md

```diff
@@ -1166,6 +1167,46 @@ std::cout << doc.find_field("k0") << std::endl; // Prints 27
 When the JSON path is the empty string (`""`) applied to a scalar document (lone string, number, Boolean or null), a SCALAR_DOCUMENT_AS_VALUE error is returned because scalar document cannot
 be represented as `value` instances. You can check that a document is a scalar with the method `scalar()`.
 
+JSON Path
+------------
+
+The simdjson library now supports a subset of [JSON Path](https://goessner.net/articles/JsonPath/) through the `at_path()` method, allowing you to reach further into the document in a single call. For now this is only supported for the On Demand approach. The subset of JSON path that is implemented is the subset that is trivially convertible into the JSON Pointer format, using `.` to access a field and `[]` to access a specific index.
+
+Consider the following example:
+
+```c++
+auto cars_json = R"( [
+  { "make": "Toyota", "model": "Camry",  "year": 2018, "tire_pressure": [ 40.1, 39.9, 37.7, 40.4 ] },
+  { "make": "Kia",    "model": "Soul",   "year": 2012, "tire_pressure": [ 30.1, 31.0, 28.6, 28.7 ] },
+  { "make": "Toyota", "model": "Tercel", "year": 1999, "tire_pressure": [ 29.8, 30.0, 30.2, 30.5 ] }
+] )"_padded;
+ondemand::parser parser;
+auto cars = parser.iterate(cars_json);
+cout << cars.at_path("[0].tire_pressure[1]") << endl; // Prints 39.9
+```
+
+For multiple JSON pointer queries on a document, one can call `at_path` multiple times.
+
+```c++
```

Comment:
Ideally, the example should appear more or less "as is" in ondemand_readme_examples so that we know that they can be copied and pasted and be valid code.

Here I think that the second example is problematic.

Reply:
But generally, try to follow our convention that code examples in the documentation appear as-is in the tests so that we know that it compiles.

Reply:
Removed second example